crypto/src/main/java/org/hyperledger/besu/crypto/AbstractSECP256.java

```diff
@@ -44,13 +45,6 @@
 /** The Abstract secp256. */
 public abstract class AbstractSECP256 implements SignatureAlgorithm {
 
-  /** The constant PRIVATE_KEY_BYTE_LENGTH. */
-  protected static final int PRIVATE_KEY_BYTE_LENGTH = 32;
-  /** The constant PUBLIC_KEY_BYTE_LENGTH. */
-  protected static final int PUBLIC_KEY_BYTE_LENGTH = 64;
-  /** The constant SIGNATURE_BYTE_LENGTH. */
-  protected static final int SIGNATURE_BYTE_LENGTH = 65;
```

Comment:
are those constants no longer used? Does this relate to this PR or is it just cleanup?

Reply:
They show up as unused in IntelliJ analysis.